modules/serverless-kn-func-emit.adoc

```diff
@@ -0,0 +1,70 @@
+[id="serverless-kn-func-emit_{context}"]
+= Emitting a test event to a deployed function
+
+You can use the `kn func emit` CLI command to emit a CloudEvent to a function that is either deployed locally or deployed to your {product-title} cluster. This command can be used to test that a function is working and able to receive events correctly.
+
+You can specify the following parameters for the emitted CloudEvent:
+
+Event type:: The type of event being emitted. You can find information about the `type` parameter that is set for events from a certain event producer in the documentation for that event producer. For example, the API server source event source may set the `type` parameter of produced events as `dev.knative.apiserver.resource.update`.
+Event source:: The event source that produced the event. This may be a URL for the event source, for example `https://10.96.0.1/`, or the name of the event source.
+Event ID:: A random unique ID that is created by the event producer.
+Event data:: The data contained in the event, if you are not sending events from an existing event source that already specifies data for the event. For example, if you are sending an event that uses the default settings, you can specify a `--data` value such as `"Hello world!"` so that the event will contain this data string. You can also specify what type of data is carried by the event, by using the `--content-type` flag.
+
+You can specify these parameters by using the `kn func emit` CLI command flags, or by specifying a file on disk that contains this information for the event:
+
```

Comment:
The very first example should probably just be `kn func emit` so the reader knows that all the stuff is optional.  Otherwise, it could be a little overwhelming :)